packages/ag-charts-community/src/chart/interaction/interactionManager.ts

```diff
@@ -73,9 +73,25 @@ const EVENT_HANDLERS = [
     'keyup',
 ] as const;
 
+export const REGIONS = {
+    TITLE: 'title',
+    SUBTITLE: 'subtitle',
+    FOOTNOTE: 'footnote',
+    LEGEND: 'legend',
+    NAVIGATOR: 'navigator',
+    PAGINATION: 'pagination',
+    ROOT: 'root',
+    SERIES: 'series',
+    HORIZONTAL_AXES: 'horizontal-axes',
+    VERTICAL_AXES: 'vertical-axes',
+    TOOLBAR: 'toolbar',
+} as const;
+
+export type RegionName = (typeof REGIONS)[keyof typeof REGIONS];
```

Comment:
Could you move this to a distinct file as regions don't fit into the responsibility of InteractionManager?

(Oli and I agreed yesterday that we should remove `region` from `BaseInteractionEvent` here in a follow-up PR)

Reply:
Yes I think this can be simplified. `BaseInteractionEvent.region` and `KeyNavEvent.region` can stay as-is for this PR. I'll rebased https://github.com/ag-grid/ag-charts/pull/1640 & resolve conflicts once this PR is merged.